.gitignore

```diff
@@ -1,10 +1,10 @@
-# Generated binary files
-/obj/
-/dnsmasq/obj
+# Object files output by the assembler
+/build
```

Comment:
We switched to a common `build` directory for the object files.

https://github.com/pi-hole/FTL/blob/7f410178f8e283a30896a3307185734d74248362/Makefile#L12